app/views/research_sessions/purpose.html.erb

```diff
@@ -3,14 +3,6 @@
   <%= render 'error_summary' %>
```

Comment:
We should delete the whole file as it is no longer being used.

Reply:
Oh yes, good spot